src/effects/mod.rs

```diff
@@ -0,0 +1,49 @@
+//! # Effects
+//! Effects contains the main functionality of Sunny
+//!
+
+mod deafen;
+mod join;
+mod leave;
+pub mod now_playing;
+mod play;
+pub mod queue;
+mod skip;
+mod stop;
+
+pub use deafen::deafen;
+pub use join::join;
+pub use leave::leave;
+pub use play::play;
+pub use skip::skip;
+pub use stop::stop;
+
+use songbird::input::Metadata;
+use std::time::Duration;
+
+/// `split_duration` splits a [`Duration`] into a (minutes, seconds) tuple
+pub const fn split_duration(d: Duration) -> (u64, u64) {
```

Comment:
Do we want these to be public? They're only used within effects/

Reply:
pub (crate) ?

Reply:
Should be fine having them private right? They're still accessible to all child modules that way

Reply:
if you only use it in children you indeed can make it private